build.gradle.kts

```diff
@@ -24,13 +25,25 @@ group = "org.jmailen.gradle"
 description = projectDescription
 
 object Versions {
-    const val androidTools = "4.2.2"
-    const val jetbrainsAnnotations = "22.0.0"
+    const val androidTools = "7.0.4"
```

Comment:
Yeah, there's probably no way around losing compatibility with prior versions of AGP since we use the plugin itself to resolve the source sets and can't easily include two in the compile.

Can you go ahead and add an Android Plugin column to the compatibility matrix?
https://github.com/jeremymailen/kotlinter-gradle#compatibility

Reply:
I pushed another commit, with a "fix" which would maintain old behavior for AGP's `4.x` users. I also added `integration-tests-android` job to existing workflow which makes sure the plugin will work with older AGP versions.

![image](https://user-images.githubusercontent.com/36954793/146669159-7d8bcbe1-3263-4267-aea5-4ccd4c16fc9d.png)

Reply:
Very nice